goal_src/levels/finalboss/green-eco-lurker.gc

```diff
@@ -40,7 +40,7 @@
   0
   -1
   ((1 (meters 999999)))
-  :bounds (static-spherem 0 0 0 5.95)
+  :bounds (static-spherem 0 0 0 5.949999809265137)
```

Comment:
hm, something has gone wrong here... I bet we forgot to use the float printer for the skelgroup definition.  fine for now though

Reply:
oops, it doesn't.